assets/dev/scss/frontend/_container.scss

```diff
@@ -91,45 +126,104 @@
+				height: auto;
+				min-height: 100%;
+				animation-name: dnd-placeholder-widget-vertical;
+				animation-fill-mode: both;
+			}
 		}
 
 		// Fix nested Containers DnD issues (ED-5424).
 		&[data-nesting-level]:not( [data-nesting-level = "0"] ) {
-			// Any placeholder which is not on first widget.
+
+			&, & > .e-container__inner {
+
+				// Any placeholder which is not on first widget.
+				> .elementor-widget-placeholder {
+					margin-inline-end: 0;
+					margin-inline-start: calc( var( --margin-start ) + var( --margin-end ) );
+				}
+	
+				// Placeholder on first widget.
+				> :not( .elementor-element ) + .elementor-widget-placeholder {
+					margin-inline-end: var( --margin-end );
+					margin-inline-start: var( --margin-start );
+				}
+			}
+		}
+	}
+
+	&.e-container--column {
+
+		& > .elementor-widget-placeholder {
+
+			// Hack to fix placeholder placement in Chrome when it's last (ED-5486).
+			&:nth-last-child( 2 ) {
+				margin-block-start: calc( 2 * var( --margin-start ) );
+				--margin-end: 0;
+			}
+		}
+
+		&, & > .e-container__inner {
+			
 			> .elementor-widget-placeholder {
-				margin-inline-end: 0;
-				margin-inline-start: calc( var( --margin-start ) + var( --margin-end ) );
+				height: var( --size );
+				// To make sure that it doesn't push & break the layout when appearing.
+				margin-block-start: var( --margin-start );
+				margin-block-end: var( --margin-end );
+				animation-name: dnd-placeholder-widget-horizontal;
 			}
+		}
 
-			// Placeholder on first widget.
-			> :not( .elementor-element ) + .elementor-widget-placeholder {
-				margin-inline-end: var( --margin-end );
-				margin-inline-start: var( --margin-start );
+		& > .e-container__inner {
+
+			// Hack to increase the dragable element size inside the `boxed width` container (ED-7867).
+			& > .elementor-html5dnd-current-element:not( .elementor-element-editable ) {
+				--spacer-size: 15px;
+				position: relative;
+
+				&:before, &:after {
+					position: absolute;
+					inset-inline: 0;
+					height: var( --spacer-size );
+					width: 100%;
+				}
+
+				&::before {
+					content: '';
+					top: calc( -1 * var( --spacer-size ) );
+				}
+
+				&::after {
+					content: '';
+					bottom: calc( -1 * var( --spacer-size ) );
+				}
+			}
+
+			& > div:last-child {
+
+				// Increase the space of the bottom placeholder (ED-7867).
+				&.elementor-widget-placeholder {
+					margin-bottom: 0;
+				}
 			}
 		}
-	}
 
-	&.e-container--column > .elementor-widget-placeholder {
-		height: var( --size );
-		// To make sure that it doesn't push & break the layout when appearing.
-		margin-block-start: var( --margin-start );
-		margin-block-end: var( --margin-end );
-		animation-name: dnd-placeholder-widget-horizontal;
-
-		// Hack to fix placeholder placement in Chrome when it's last (ED-5486).
-		&:nth-last-child( 2 ) {
-			margin-block-start: calc( 2 * var( --margin-start ) );
-			--margin-end: 0;
+		&.e-container--width-boxed {
+
```

Comment:
@StyleShit 

Ideally I would like to find a JS solution for this. 

Currently the `.elementor-widget-placeholder` is inserted in the following places: 

.e-container--width-boxed 
.e-container--width-boxed > .e-container__inner
.e-container--width-boxed > .e-container__inner > .elementor-element

I would like to remove the placeholder from `.e-container--width-boxed` , but I haven't found a JS solution for this.

Currently the editor creates in some cases multiple placeholders on drag.